draft-ietf-rats-corim.md

```diff
@@ -1109,6 +1109,10 @@ Endorsed Value Claims, the subject is either a Target or Attesting Environment,
 the object contains measurements, and the predicate defines semantics for how
 the object relates to the subject.
 
+Said another way, take the pair of `environment-map` and `measurement-map / mkey` from a `endorsed-triple-record`.
+If this same pair matches the pair from a `reference-triple-record` from the same CoMID, consider that record to be an implicit condition for adding the `endorsed-triple-record` to the ACS.
```

Comment:
The way I interpret this is different (apologies in advance for the poetic license).
If I am given some endorsed value $$EV_{i} = [ Env_{j}, Measurement_{k} ]$$
and the ACS's state is currently $$ACS = [ Env_{j}, Measurement_{l} ]$$
then $$Measurement_{k}$$ can be added to the ACS:
$$ACS = [ Env_{j}, Measurement_{l}, Measurement_{k} ]$$

Reply:
So the ACS has to have Some measurement by Somebody, irrespective of the reference values in the CoMID? You have no requirements on Measurement_l other than it exists (and therefore non-empty).

Reply:
> So the ACS has to have Some measurement by Somebody irrespective of the reference values in the CoMID?

The ACS is initialised with evidence (see [Section 8.4.1](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#section-8.4.1-1)).  At least, there's one environment ($$Env_{j}$$) and one measurement associated with that ($$Measurement_{l}$$).

Reply:
Endorsements have much greater degree of flexibility.

There can be `one` type of Endorsement belonging to an Attester Class, (identified by Class-ID) regardless of what Reference Value holds for the component!

Reply:
> additional measurements **that are valid when** a Target Environment **has been verified against reference measurements**

Yes, this looks to me like prose that has been obsoleted by the recent work on the appraisal phases.  It should be changed accordingly.